src/overlays/actors/ovl_En_Ossan/z_en_ossan.c

```diff
@@ -1342,12 +1342,12 @@ void EnOssan_InitPartTimeWorker(EnOssan* this, GlobalContext* globalCtx) {
 
 s32 EnOssan_GetWelcomeCuriosityShopMan(EnOssan* this, GlobalContext* globalCtx) {
     Player* player = PLAYER;
-    u16 ret = func_800F1250(globalCtx, 0x2F);
+    u16 faceReaction = Text_GetFaceReaction(globalCtx, 0x2F);
```

Comment:
This function returns a text id, so I suggest `textId` for the names here.

Reply:
Oh I didn't even notice that. I thought it was returning some index into a facial expression array or something. In that case, yeah, that sounds like a better name than what I suggested. Both are probably better than the non-descriptive "ret" though.

Reply:
I'm on it...